### ISSUE TITLE
Git Basics: <Accessibility of screenshots>

### ISSUE THREAD

**Issue by MatthewWestDev** [TheOdinProject/curriculum#27904]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

I'm a low-vision user and this is the first page that I had some difficulty receiving the lesson information due to heavily relying on screenshots. In step 2 of Create the Repository, I am directed to click the button shown in the screenshot. I was able to enlarge the image quite a bit to get the information but a written description of the button itself and where it is located would be helpful. In the rest of the lesson I was unable to use the screenshots of the shell outputs to compare to my local output. It would be some work but it would greatly help if actual text was used instead of the screenshot. Most steps included key information of what to look for in my output which was helpful to confirm I was on the right track. My suggestion moving forward is to use text instead of screenshots if the screenshot is essentially text. Thanks!

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-git-basics

**Comment by MaoShizhong** [comment]
Thank you for opening the issue @MatthewWestDev!
Accessibility is definitely something we're steadily working on improving throughout the site and curriculum.

@thatblindgeye I think we'd greatly value your thoughts here

**Comment by thatblindgeye** [comment]
We definitely do rely too heavily on screenshots to convey important information. Where providing a screenshot is helpful we should also provide a text alternative (beyond just `alt` text, because that will only be visible for users navigating via AT such as screen readers or only when the image fails to load). Otherwise, yes we should replace screenshots with text. Relying on screenshots so much also just makes it even more upkeep if even the slightest thing changes on the UI vs the screenshot ("Well now there's another field between these two fields from the screenshot")

I'll open this as help wanted for now, but may try to get around to it myself if I can/nobody else asks to do it.

**Comment by kol3x** [comment]
Looks like there is a lot of work to do here. I'd like to take it, but don't know if I will have time in the coming days.
